test/components/creeps/creepActions.test.ts

```diff
@@ -0,0 +1,36 @@
+/// <reference path="../../../typings/globals/screeps/index.d.ts"/>
```

Comment:
Could this be done without using `/// <reference path="..." />` as per the default `tslint` rules?